include/details.md

```diff
@@ -42,18 +43,22 @@ __Documentation__
     - {{ feature }}
     {% endfor %}
 
-{% if (features.programming_languages or features.environments) %}
-=== "Kernels"
+{% if (features.programming_languages or features.environments) %} 
+=== "Kernels :material-information-outline:{ title="Links to Kernels" }" 
     {% if features.programming_languages %}
     Programming Languages:
     {% for kernel in features.programming_languages %}
+    {% if Links[kernel] %}
+    - [{{ kernel }}]({{ Links[kernel] }})
+    {% else %}
     - {{ kernel }}
-    {% endfor %}
     {% endif %}
-    {% if features.environments %}
+    {% endfor %}
+{% endif %}
+    {% if features.environments %} 
     Environments:
     {% for kernel in features.environments %}
-    - {{ kernel }}
+    - {{ kernel }} {% if Links[kernel] %} {{ Links[kernel] }} {% endif %}
```

Comment:
No else-part here?

Reply:
I have added an else part that will display hyperlink if an environment is found else it will just display the name of the environment setup in plain text.